client.go

```diff
@@ -37,16 +43,16 @@ func NewClient(c Config) (*Client, error) {
 	}
 
 	return &Client{
-		client: goredis.NewClient(&goredis.Options{
-			Addr:     c.Addr,
+		client:  RedisClient{client:redis.NewClient(&redis.Options{
+			Addr: c.Addr,
 			Password: c.Password,
-			DB:       c.Database,
-		}),
-		ttl: c.TTL,
+			DB: c.Database,
+		})},
+		ttl:     0,
```

Comment:
Consider adding an error if ttl is 0. I assume that TTL 0 means the entry will expire immediately? Or never expire which I doubt is functionality we want. I think it would better to error or apply a sensible non 0 default

Reply:
Added a check for ttl being 0. Also added new test to cover it. With the default value I'll leave that for the configuration in the sessions api.